parse/parse.go

```diff
@@ -409,7 +410,7 @@ func (p *parser) parseExpression() (Expression, error) {
 
 		return errorf("%w: %w", mf2.ErrBadOperand, err)
 	case itemVariable:
-		variable := Variable(itm.val)
+		variable := Variable(norm.NFC.String(itm.val))
```

Comment:
Can you please add a comment to `Variable` that it contains a NFC normalised name?